app/views/food_orders/show.html.erb

```diff
@@ -1,18 +1,24 @@
 <%# ------------ if cart is empty %>
 
-
-<h1 class="no-items d-none cart-title text-muted text-center">Your cart is empty</h1>
-
+<div class="empty-cart-container no-items d-none">
+  <h1 class="cart-title text-muted">Your cart is empty</h1>
+</div>
 
 <%#  ----------- when there is something in the cart %>
 
 <div id="cart" class="d-none" style="background-color: white">
 
-  <div class="card-items">
-    <h1 class="mt-3"> Restaurant name </h1>
-    <p> Food items</p>
+  <div style="background-color: white">
+    <div class="container py-3">
+      <h1 class="mt-3">Last Step - Checkout</h1>
+      <p class="checkout-restaurant-name"></p>
```

Comment:
what is this empty p tag doing?

Reply:
it's for me to do innerText that inserts the resto name in JS

Reply:
tagged you in the relevant line